src/test/scala/com/mowczare/kafka/streams/example/stream/FrequencyStreamTest.scala

```diff
@@ -0,0 +1,57 @@
+package com.mowczare.kafka.streams.example.stream
+
+import com.madewithtea.mockedstreams.MockedStreams
+import com.mowczare.kafka.streams.example.model.InputEvent
+import com.mowczare.kafka.streams.example.serde.SerdeUtil
+import com.mowczare.kafka.streams.pds.frequency.ItemSketchWrap
+import com.yahoo.sketches.frequencies.ErrorType
+import org.apache.kafka.streams.scala.Serdes
+import org.scalatest.{FunSuite, Matchers}
+
+class FrequencyStreamTest extends FunSuite with Matchers {
+
+  val admissibleError = 0.1
+
+  val inputTestTopic = "input-test"
+  val outputTestTopic = "output-test"
+
+  def almostEqual(result: Double, shouldBe: Double): Boolean = {
+    Math.abs(result - shouldBe) / shouldBe < admissibleError
```

Comment:
This implementation never works with negative `shouldBe` values, please move the `)` a bit to the right.

Reply:
Imo in this scope it is irrelevant, but ok.